src/Sylius/Behat/Page/Admin/Country/CreatePage.php

```diff
@@ -11,14 +11,69 @@
 
 namespace Sylius\Behat\Page\Admin\Country;
 
+use Behat\Mink\Driver\Selenium2Driver;
 use Sylius\Behat\Behaviour\ChoosesName;
 use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
  */
 class CreatePage extends BaseCreatePage implements CreatePageInterface
 {
     use ChoosesName;
+
+    /**
+     * @var int
+     */
+    private $provincesCount = 0;
+
+    /**
+     * @var array
+     */
+    protected $elements = [
+        'provinces' => '#sylius_country_provinces',
+    ];
+
+    /**
+     * If the scenario is using a "@javascript" tag we are selecting Country name differently
+     *
+     * @param string $name
+     */
+    public function selectName($name)
+    {
+        $driver = $this->getSession()->getDriver();
```

Comment:
```php
if ($this->getSession()->getDriver()instanceof Selenium2Driver) {
    $this->chooseNameWithJavascript($name)
}
```
WDYT? It would be much more readable